packs/equipment/bracers-of-strength.json

```diff
@@ -46,7 +46,7 @@
             },
             {
                 "key": "FlatModifier",
-                "label": "Belt of Giant Strength (lift heavy objects, Escape, Force Open)",
+                "label": "Bracers of Strength (lift heavy objects, Escape, Force Open)",
```

Comment:
This should be either localized, or removed with a slug added.

Reply:
The slug isn't optional if there's any overlap in selectors, even if the type is different.